p2p/ipld/plugin/go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/lazyledger-core/p2p/ipld/plugin
```

Comment:
This should instead be named:

`module github.com/lazyledger/lazyledger-core/p2p/ipld/plugin`